README.md

```diff
@@ -75,7 +75,7 @@ command line).
 Now you can build it with:
 
 ```console
-make
+make -j8
```

Comment:
We don't need more than 9 threads because there are totally 9 C++ source files. Each compiler thread starts compiling from a single source file.
In addition, 8 sounds nice if their processors are 4 cores/8 threads.

Reply:
This is true in case the system supports many threads. However, when developing on a laptop (which is very common), some systems only support 2 threads. In this case, using -j8 would result in performance degradation.

Reply:
4-cores-8-threads processors become popular since the 8th gen Intel Core processors release (mainstream level processors, e.g. i5 8250U). It's already 5 years since that so I think most students will have a 4-core-8-thread processor or better.

BTW, I also tested the compiling time on the login node with different numbers of threads:
-j1:  wall time-15.5s, CPU time-11.419s
-j2:  wall time-8.197s, CPU time-11.465s
-j4: wall time-5.567s, CPU time-11.811s
-j8: wall time-5.850s, CPU time-11.864s
-j16: wall time-6.389s, CPU time-12.788s
Maybe -j4 is a better choice.

Reply:
I agree .. I think testing compile time is the best way to find the optimal "-j" value. Can you test also on login2 node?